examples/kubernetes/connectivity-check/main_tool.cue

```diff
@@ -75,8 +76,20 @@ ccCommand: {
 		}
 	}
 
+	task: filterTraffic: {
+        if #flags.traffic == "internal" {
+            resources: [ for x in task.filterName.resources if x.metadata.labels.traffic != "external"  {x}]
+        }
+		if #flags.traffic == "external" {
+            resources: [ for x in task.filterName.resources if x.metadata.labels.traffic != "internal"  {x}]
+        }
```

Comment:
you could probably simplify this to one if condition:
```suggestion
        if #flags.traffic != "any" {
            resources: [ for x in task.filterName.resources if x.metadata.labels.traffic != #flags.traffic  {x}]
        }
```

Reply:
should it be something like below as x.metadata.labels.traffic could be "any"? We also need another default case to avoid ` undefined field "resources"` issue.

```
		if #flags.traffic != "any" {
			resources: [ for x in task.filterName.resources if x.metadata.labels.traffic == #flags.traffic || x.metadata.labels.traffic == "any" {x}]
		}
		if #flags.traffic == "any" {
			resources: task.filterName.resources
		}
```